how-to/configure-celery.rst

```diff
@@ -12,8 +12,8 @@ How to configure Celery
 =======================
 
 
-This article assumes that you are already familiar with the basics of using Celery with Django and that you have Celery installed
-in your application.
+This article assumes that you are already familiar with the basics of using Celery with Django and that you have Celery
+installed in your application.
```

Comment:
If you can avoid introducing formatting changes, it's better. Otherwise the reviewer has to read through irrelevant changes.